CHANGELOG.md

```diff
@@ -8,6 +8,12 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
 
+### Changed
+
+- Use first LMS Backend settings for authentication.
```

Comment:
```suggestion
- Use first item in LMS Backend settings for authentication.
```

Reply:
Not a big fan of this pattern. I think we should have separate settings for what is authentication and for what is API bridge.

Reply:
I agree. I create an AUTHENTICATION_BACKEND setting.